SU-HSDO/suhumsci
# STN-714: Backstop.js

# READY FOR REVIEW

## Summary
This pull requests removes staging basic auth information from our backstop.js config. Username and password will now live in a .env file that is git ignored. 

## Need Review By (Date)
asap

## Urgency
high

## Steps to Test
In the Humsci Basic directory:
1. Delete your node_modules directory
2. Run `npm i`
3. Make a copy of the `.env-sample` called `.env` and add the basic auth credentials found in 1Pass
4. Run `npm run backstop: test` to generate the `backstop.json` file in the backstop directory
5. Check for the following:
    - [x] The `.env` file is ignored by git
    - [x] The `backstop/backstop.json` file is ignored by git
    - [x] You can still run `npm run backstop:reference` and `npm run backstop:test` to run regression testing
    - [x] `npm start` still runs the project as expected

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
